scripts/change.md

```diff
@@ -0,0 +1,27 @@
+{%- macro change(c) -%}
```

Comment:
I'm sure I've asked this before, but it bothers me that the tera template file is not actually a Markdown file, despite having a `.md` extension. Is this some requirement of Tera, or could we avoid that?